pkg/server/server.go

```diff
@@ -811,6 +812,17 @@ func (s *Server) Start(ctx context.Context) error {
 	if err != nil {
 		return errors.Wrap(err, "inspecting engines")
 	}
+	if runtime.GOOS != "windows" {
+		timer := time.NewTimer(time.Second * 30)
```

Comment:
golf: use [`time.AfterFunc`](https://godoc.org/time#AfterFunc)
```golang
defer time.AfterFunc(30 * time.Second, func() {
		log.Shout(context.Background(), log.Severity_WARNING,
				"Server is timing out while waiting for other nodes in the cluster.\n\n"+
					"Please restart the other nodes without waiting for this one to finish. \n"+
					"One way to do this is by appending & to your command instead of using --background. \n")
}).Stop()
```
(note that the Stop is what's being deferred)